index.html

```diff
@@ -66,14 +66,191 @@ <h1>I’m Tracy<br />Glad to see,You!</h1>
     </section>
     <section class="grid-container">
       <div class="item1">
-        <div class="item1a"></div>
+        <div class="item1a">
+          <img src="./images/snapshoot.png" />
+        </div>
+        <div class="item1b">
+          <h1>Tonic</h1>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item2">
+        <div class="item2a">
+          <img src="./images/snap2.png" />
+        </div>
+        <div class="item1b">
+          <h1>Multi-Post Stories</h1>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
```

Comment:
- [ ] Good job adding the languages with the `ul` tag :clap: , however, as mentioned previously you should make sure the design you get on this part stick to the Figma design. It means the padding of these languages should be equal to the one used in the Figma design :+1: You did a great effort adding the padding top and bottom but I what do you think  about adding a padding left and right ? :smile: 
Your Design


![Screenshot from 2023-01-13 16-28-10](https://user-images.githubusercontent.com/80458199/212343951-2e93b7d6-0a33-4aff-91f4-15945af478b1.png)

Figma design

![Screenshot from 2023-01-13 14-49-27](https://user-images.githubusercontent.com/80458199/212324431-e3d7572d-4eb6-44ea-8ddb-42c3a43253ff.png)